docs/content/observatorium/thanos/README.md

```diff
@@ -1,15 +1,28 @@
-# Thanos Access
+# Interacting with Operate First Thanos data
```

Comment:
Can we keep the title short, as this is converted to a navbar title: 

![image](https://user-images.githubusercontent.com/10904967/144609959-a17edf76-b6b8-4dd8-9a1e-cc3689dfe7f3.png)

Reply:
updated :+1: